classes/Tools.php

```diff
@@ -743,6 +743,10 @@ public static function convertPriceFull($amount, Currency $currency_from = null,
             $currency_from = new Currency(Configuration::get('PS_CURRENCY_DEFAULT'));
         }
 
+        if ($currency_to === null) {
```

Comment:
Could you, please, use [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions)? 
`null === $currency_to`